### PR TITLE
Applying styleguide.

### PR DESCRIPTION
**Pull request by Irsanarisandy**
### DoD (Definition of Done) Checklist

- [x] There are no outstanding high priority defects or regressions
- [x] Feature is tested in agreed devices and latest version of browsers**
- [x] The developer is confident that Acceptance Criteria have been met
- [x] Branch rebased against `develop`
- [x] Non-MVP enhancements have been captured in another card
- [x] Add `ngrok` link to site

##### * Design DOD
- [x] The design signed off by Beth (if appropriate)*
- [x] Accurately matches design unless there is a good reason not to
- [x] Hover states & interactions are correct
- [x] No regressions of other styles

##### ** Agreed Browsers and Devices
- **Browsers** - Chrome (Development Team), IE, Firefox, Safari
- **Devices** - iPad Pro, Surface Tablet.

### Peer Review Checklist
- [x] Must be peer reviewed by one of the team members first before passing it onto mentor
- [x] Manually tested
- [x] Runs without issue locally
- [x] Browser/Device tested
- [x] Code is quality reviewed
- [x] Feedback is provided and documented
- [x] Matches all Acceptance Criteria

### CODE REVIEW

File: themes/app/src/scss/_typography.scss [hunk 1]
```diff
@@ -1,12 +1,160 @@
...
+    line-height: 1.2;
+    margin: 8px 0;
+}
+
+h2 {
+    font-family: 'SourceSansPro', sans-serif;
+    font-weight: 400;
+    font-size: 28px;
+    color: $navy-blue;
+    line-height: 1.2;
```
Comment (IshaqIdris) [inline themes/app/src/scss/_typography.scss:45]: Could be made into a line weight variable for easy changes in the future :)

File: themes/app/src/scss/_typography.scss [hunk 2]
```diff
@@ -1,12 +1,160 @@
...
+    margin: 8px 0;
+}
+
+h3 {
+    font-family: 'SourceSansPro', sans-serif;
+    font-weight: 600;
+    font-size: 26px;
+    color: $navy-blue;
+    line-height: 1.3;
```
Comment (IshaqIdris) [inline themes/app/src/scss/_typography.scss:54]: Same with these line weight variables `1.3`

File: themes/app/src/scss/_typography.scss (hunk 1 above)
Comment (Irsanarisandy) [reply themes/app/src/scss/_typography.scss:45]: Not for styleguide, but will keep that in mind next time for others 😃

File: themes/app/src/scss/_typography.scss (hunk 2 above)
Comment (Irsanarisandy) [reply themes/app/src/scss/_typography.scss:54]: Not for styleguide, but will keep that in mind next time for others 😃

**Pull request merged**